src/ngResource/resource.js

```diff
@@ -6,6 +6,18 @@ var $resourceMinErr = angular.$$minErr('$resource');
 // stopping at undefined/null.  The path must be composed of ASCII
 // identifiers (just like $parse)
 var MEMBER_NAME_REGEX = /^(\.[a-zA-Z_$@][0-9a-zA-Z_$@]*)+$/;
+var PROTOCOL_AND_IPV6_REGEX = /^https?:\/\/\[[^\]]*][^/]*/;
+var ESCAPED_COLONS_REGEX = /\\:/g;
+var ONLY_DIGITS_REGEX = /^\d+$/;
+var encodeUriQuery = angular.$$encodeUriQuery,
```

Comment:
You actually want to do that once you know AngularJS has been loaded (e.g. [here][1]), because at this point it might be the case that only `angular-loader` has been loaded 😱 

[1]: https://github.com/angular/angular.js/pull/16479/files#diff-67ae87630c0e91cba343677ed730daafR721